tests/test_segmenter.py

```diff
@@ -46,6 +46,16 @@ def test_exception_with_both_clean_and_span_true():
     assert str(e.value) == "char_span must be False if clean is True. "\
                             "Since `clean=True` will modify original text."
 
+def test_exception_with_otherthan_en_lang_span_true():
```

Comment:
Is it possible to support the `char_span` functionality for any language with a corresponding spacy model?

Reply:
Removed dependency on spacy in this [recent commit](https://github.com/nipunsadvilkar/pySBD/pull/67/commits/7d75901742ec5b82f71558f39e8243da90976582)